tests/helpers/relationship.js

```diff
@@ -3,25 +3,25 @@ import Ember from 'ember';
 
 const { get } = Ember;
 
+function runAssertions(assert, store, modelName, relationshipName, relationshipKind) {
```

Comment:
Thoughts on renaming this function to better reflect that it is looking for relationships? `runAssertions` seems pretty broad.